runmariadb.sh

```diff
@@ -3,6 +3,18 @@ PATH=$PATH:/usr/sbin/
 DATADIR="/var/lib/mysql"
 MARIADB_PASSWORD=${MARIADB_PASSWORD:-"change_me"}
 MARIADB_CONF_FILE="/etc/my.cnf.d/mariadb-server.cnf"
+MARIADB_CERT_FILE=/certs/mariadb/tls.crt
+MARIADB_KEY_FILE=/certs/mariadb/tls.key
+
+mkdir -p /certs/mariadb
+if [ -f "$MARIADB_CERT_FILE" ] && [ ! -f "$MARIADB_KEY_FILE" ] ; then
+    echo "Missing TLS private key file /certs/mariadb/tls.key"
+    exit 0
```

Comment:
If there's ever a time for `exit 1` surely this is it.

Reply:
Fixed! Thanks